sirepo/pkcli/hellweg.py

```diff
@@ -12,7 +12,7 @@
 from sirepo.template import template_common
```

Comment:
Two lines up (line 10), change to:

```py
import rshellweg.solver
```

It's only used once so it doesn't need to be unqualified.

Also remove the `from __future__`